conda_build/license_family.py

```diff
@@ -14,11 +13,11 @@
 GPL
 BSD
 MIT
-APACHE
+Apache
 PSF
-PUBLICDOMAIN
-PROPRIETARY
-OTHER
+Public-Domain
```

Comment:
> I generally advise you to not set license family manually. If you have strong reason to, we should talk more.

Ah, thanks, I did not know that, I thought it was recommended to specify it (maybe because it is included in conda-forge's example `meta.yaml`). In my case, I was setting it to `Other` which `guess_license_family` will do automatically, anyway.